databases/elasticsearch/Portfile

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                elasticsearch
+version             6.1.1
+revision            1
+
+categories          databases java
+maintainers         nomaintainer
```

Comment:
new ports should have a maintainer

Reply:
The OP said he can't commit to being the maintainer, though he could always put himself and "openmaintainer" down.